### DISCUSSION TITLE
How to call `SuggestedWarpOutput()`?

### DISCUSSION THREAD

**Discussion by netthier** [MaxRev-Dev/gdal.netcore#174] [Q&A] [answered]
Hello, I'm relatively new to C# and this library, so I'm having difficulties finding which GDAL C/C++ functions are available.
I would like to call [GdalSuggestedWarpOutput](https://gdal.org/en/stable/api/gdal_alg.html#_CPPv423GDALSuggestedWarpOutput12GDALDatasetH19GDALTransformerFuncPvPdPiPi) which seems to have a swig-wrapper here as well: https://github.com/OSGeo/gdal/blob/master/swig/include/Operations.i#L828
However, I've been unable to find this function using the tools offered by my IDE. Am I just looking in the wrong places or is this function not (yet?) available in C#?

**Comment by MaxRev-Dev** [comment, marked as answer]
It's available as SuggestedWarpOutput

### Rider
<img width="1639" alt="image" src="https://github.com/user-attachments/assets/111c5e77-0e88-495d-af2b-2ed095d2e48f" />

### VSCode
<img width="1070" alt="image" src="https://github.com/user-attachments/assets/8d8e7aec-fd46-4e92-b2e8-292c2566aa26" />

**Reply by netthier** [reply]
Ah, so sorry, it turned out I was using an older version of `MaxRev.Gdal.Core`. I've now upgraded to the latest one and the function is shown as expected :sweat_smile: 
In my defense, I didn't set up the project :^)
